src/database/index.tsx

```diff
@@ -62,17 +62,27 @@ const authenticate = async () => {
 };
 
 const currrentUserId = () => {
-  return (firebase.auth().currentUser || {}).uid;
+  const userId = (firebase.auth().currentUser || {}).uid;
+
+  if (!userId) {
+    throw new Error("currrentUserId() always expects a user to be returned");
+  }
+
+  return userId;
 };
 
-const getInputModelsDocRef = async () => {
+const getDb = async () => {
```

Comment:
Sorry, I normally like to include refactoring/cleanup code in separate PRs, but we're moving fast!